controllers/cstates_controller.go

```diff
@@ -148,7 +160,7 @@ func (r *CStatesReconciler) verifyCSStatesExist(cStatesSpec *powerv1.CStatesSpec
 		}
 	}
 	for coreID, coreCStates := range cStatesSpec.IndividualCoreCStates {
-		logger.V(5).Info("checking the power library to confirm that C-States are a individual core", "coreID", coreID)
+		logger.V(5).Info("checking the power library to confirm that C-States are on individual core", "coreID", coreID)
```

Comment:
```suggestion
		logger.V(5).Info("checking the power library to confirm that C-States are on individual cores", "coreID", coreID)
```

Reply:
Will address all changes to controllers/controller tests in the upcoming release